lma-main.yaml

```diff
@@ -502,7 +502,7 @@ Parameters:
 
   AmazonQAppId:
     Type: String
-    AllowedPattern: "^[a-zA-Z0-9][a-zA-Z0-9-]{35}$"
```

Comment:
Thanks Austin! Rather than remove the pattern completely, a better way is to use a pattern that allows the value to be blank, or comply with a regex..  See how we did it for BedrockKnowledgeBaseID [here](https://github.com/aws-samples/amazon-transcribe-live-meeting-assistant/blob/main/lma-main.yaml#L50).  It's all good though.. I'll merge and proceed.. Thanks again!